gaffer-core/operation/src/main/java/gaffer/operation/AbstractValidatable.java

```diff
@@ -126,6 +126,15 @@ protected Builder(final OP_TYPE op) {
          * @return this Builder
          * @see gaffer.operation.Validatable#setElements(Iterable)
          */
+        public Builder<OP_TYPE, OUTPUT> elements(final Element... elements) {
+            return elements(Arrays.asList(elements));
```

Comment:
This could throw a NullPointer as the method can be called with no args or a null object and this would throw a NullPointer after Arrays.asList